go.mod

```diff
@@ -1,22 +1,16 @@
 module github.com/terraform-providers/terraform-provider-mongodbatlas
 
-go 1.13
+go 1.14
```

Comment:
@PacoDw fyi^.  Let me know on this.

Reply:
It isn't required, but it the last golang version and I put it there to consider it what do you think @marinsalinas? and yes I will change the README file in this same PR, good catch 👍

Reply:
I order to reduce complexity in troubleshooting PRs should be separated in my experience, the bug fix in it's own PR/version upgrades in another.  Otherwise if you have any side issues then one can't be sure what caused it when more than one factor changes.   Just keeps things more simple

Reply:
Totally agree :)